mysql/tests/compose/mariadb.yaml

```diff
@@ -13,6 +13,8 @@ services:
       - MARIADB_DATABASE=my_database
     ports:
       - "${MYSQL_PORT}:3306"
+    volumes:
+      - ${MYSQL_CONF_PATH}:/bitnami/mariadb/conf/my_custom.cnf:ro
```

Comment:
Note to reviewers: the path to this config file is officially documented [here](https://github.com/bitnami/bitnami-docker-mariadb#configuration-file).